docs/topics/auth/passwords.txt

```diff
@@ -693,6 +693,40 @@ or if you have API calls that allow passwords to be set, for example.
 Note that where the password is passed to one of these functions, this should
 always be the clear text password - not a hashed password.
 
+If you would like to override the default error messages if a password fails
```

Comment:
I think instead of adding this here, I would document the hooks

To get you started:
```diff
     Validates that the password is of a minimum length.
     The minimum length can be customized with the ``min_length`` parameter.
 
+    .. method:: get_error_message()
+
+        .. versionadded:: 5.2
+
+        A hook for customizing the ``ValidationError`` error message. Defaults
+        to ...
+
 .. class:: UserAttributeSimilarityValidator(user_attributes=DEFAULT_USER_ATTRIBUTES, max_similarity=0.7)
```

Reply:
OK, I documented the hooks.